### PR TITLE
update added no reset theme

### PR DESCRIPTION
**Pull request by VideoGameRoulette**

### CODE REVIEW

File: frontend/src/components/ThemedWindow.jsx [hunk 1]
```diff
@@ -130,6 +147,7 @@ const ThemedWindow = ({ children, variant, slim, className: externClassName }) =
       [classes.gdq2]: search.has('gdq2'),
       [classes.gdq3]: search.has('gdq3'),
       [classes.avsr]: search.has('avsr'),
+      [classes.avsr]: search.has('noreset'),
```
Comment (markrawls) [inline frontend/src/components/ThemedWindow.jsx:150]: `classes.noreset` I think is what you want?

Comment (VideoGameRoulette) [reply frontend/src/components/ThemedWindow.jsx]: fixed

File: frontend/src/components/ThemedWindow.jsx [hunk 2]
```diff
@@ -85,6 +87,11 @@ const useStyles = makeStyles(theme => ({
         borderImage: `url(${AVSRBorder}) 9 round`,
         border: '9px solid transparent',
       },
+      '&$noreset': {
+        borderImage: `none 9 round`,
```
Comment (markrawls) [inline frontend/src/components/ThemedWindow.jsx:91]: `borderImage: 'none'` is more semantically correct for that

Comment (VideoGameRoulette) [reply frontend/src/components/ThemedWindow.jsx]: done

**Pull request merged**